src/components/ConfirmedDeletionButton/styles.tsx

```diff
@@ -0,0 +1,37 @@
+import { makeStyles, Theme as AugmentedTheme } from '@material-ui/core/styles';
+
+export default makeStyles(
+  (theme: AugmentedTheme) => ({
+    container: {
+      display: 'inline-block'
+    },
+    paper: {
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+      borderRadius: '10px',
+      border: '5px solid rgba(74, 74, 74, 0.5)',
+      backgroundColor: theme.palette.grey['800'],
+      color: 'white',
```

Comment:
I updated the way we handle colors in my PR. You should be using `theme.palette.common` for these colors. I think to get that to work I had to add a couple lines to `theme.tsx` though

Reply:
What do we think about making this an item in the backlog to go through and update all colors?

Reply:
I just merged my PR so if you rebase you will have the updated palette